components/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
@parush-uppal could we please change the folder structure to `/components/Button/index.js` do that we nest the button component inside a folder called `Button`. 

This is to enable us write tests for that component later on.

Reply:
@parush-uppal instead of writing the text prop as `t` could we change it to `text` so as to enable easy code readability.

Reply:
could the following props names be changes 

- `t` to `text`
- `tc` to `textColor`
- `bc` to `backgroundColor`

this is to ensure easy code readability

Reply:
Did the naming changes in the props just wanted to know about button component do you want me to create a separate button folder inside components or want me to move button folder inside pages